frontend/app/components/profile/profile.tsx

```diff
@@ -32,21 +32,32 @@ async function signout() {
 export function Profile() {
   const intl = useIntl();
   const rootRef = useRef<HTMLDivElement>(null);
+  const commentsLimit = useMemo(() => 10, []);
   const user = useMemo(() => parseQuery(), []);
   const [isCommentsLoading, setIsCommentsLoading] = useState(false);
   const [error, setError] = useState(false);
   const [comments, setComments] = useState<CommentType[] | null>(null);
   const [commentsAmount, setCommentsAmount] = useState<number | null>(null);
+  const [commentsSkipCounts, setCommentsSkipCounts] = useState<number>(0);
   const [isSigningOut, setSigningOut] = useState(false);
+  const isLoadMoreVisible = commentsAmount && commentsAmount > commentsSkipCounts + commentsLimit;
+
+  const fetchUserComments = useCallback(
+    async (skip: number = 0) => {
+      const { comments, count } = await getUserComments(user.id, { skip, limit: commentsLimit });
+      return { comments, count };
+    },
+    [user.id, commentsLimit]
+  );
 
-  async function fetchUserComments(userId: string) {
+  const fetchUserCommentsOnMount = useCallback(async () => {
```

Comment:
It could be moved inside `useEffect` because it is used only there and this function will be called only once.

Reply:
@akellbl4  I created this function to underline separation between `fetchUserCommentsOnMount` and `fetchMoreUserComments`. In opinion it increases readability of the component, but I can be wrong :D Anyway, please let me know if you still prefer to put the body of the function to `useEffect`, I'll do it.

Reply:
Hm. I checked the code a bit closer.

`fetchUserCommentsOnMount` and `fetchMoreUserComments` are nearly identical.
The only difference in this `setComments([...(comments || []), ...nextComments]);`. As I understand we can use only one function for both cases and we wont need to have `fetchUserComments`.